app/src/features/Incentives/components/IncentiveSectionHeader/index.scss

```
@@ -0,0 +1,50 @@
+.incentive-section-header-container {
+  &_title-container {
```

Just say `incentive-section-header-container` for outer container and let the inner class be `title-container` instead of `incentive-section-header-container_title_container`
```suggestion
.incentive-section-header-container {
  &_title-container {
```